hack/central-repo/upload-plugins.sh

```diff
@@ -275,7 +275,6 @@ cat $ROOT_DIR/create_tables.sql | sqlite3 -batch $database
 for name in ${globalPlugins[0]}; do
     addPlugin $name global true v0.0.1
     addPlugin $name global true v9.9.9
-    addGroup vmware tkg v9.9.9 $name global v9.9.9
```

Comment:
Before this change only the v9.9.9 version of global plugins were getting added to the `vmware-tkg/v9.9.9` group but not v0.0.1 plugins. Removed this to be consistent across versions.

Reply:
But then we don't have any group that contains global plugins, which may be useful to have.
How about adding the v0.01 instead of removing the 9.9.9 one?

Reply:
> But then we don't have any group that contains global plugins, which may be useful to have.
It doesn't matter for air-gapped testing. We need to have plugins that are not part of any plugin group for writing tests.